src/steps/update_horizon_scores.py

```
@@ -46,6 +46,59 @@ class HorizonScoringError(Exception):
 ANTHROPIC_API_URL = "https://api.anthropic.com/v1/messages"
 
 
+class StateAdapter:
+    """Provide safe state backed by a Pipedream Data Store when one is wired."""
+
+    def __init__(self, pd):
```

Ruff (0.16.1)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)